src/etc/featureck.py

```diff
@@ -78,8 +78,15 @@
         if not filename.endswith(".rs"):
             continue
 
+        if sys.version_info.major == 2:
+            _open = lambda f: open(f, 'r')
+        elif sys.version_info.major == 3:
+            _open = lambda f: open(f, 'r', encoding="utf-8")
```

Comment:
Is there a way to open a file as UTF-8 in python2? All our source files should be valid UTF-8 I believe at least